### PR TITLE
SAK-32240 Access to the specific message from email footer

### PR DESCRIPTION
**Pull request by juanjmerono**

### CODE REVIEW

File: msgcntr/messageforums-app/src/java/org/sakaiproject/tool/messageforums/PrivateMessagesTool.java [hunk 1]
```diff
@@ -1457,6 +1457,35 @@ public void initializeFromSynoptic()
 	  }
   }
   
+  public String getReceivedTopicForMessage(String msgId) {
+	  if (msgId!=null && getPvtAreaEnabled()) {
+		  for (int i = 0; i < pvtTopics.size(); i++) {
```
Comment (ern) [inline msgcntr/messageforums-app/src/java/org/sakaiproject/tool/messageforums/PrivateMessagesTool.java]: Use newer for loop unless you need access to the index.

File: msgcntr/messageforums-app/src/java/org/sakaiproject/tool/messageforums/PrivateMessagesTool.java [hunk 2]
```diff
@@ -1457,6 +1457,35 @@ public void initializeFromSynoptic()
...
+			  Topic topic = (Topic)pvtTopics.get(i);
+			  String typeUuid = getPrivateMessageTypeFromContext(topic.getTitle());
+			  decoratedPvtMsgs = prtMsgManager.getMessagesByType(typeUuid, PrivateMessageManager.SORT_COLUMN_DATE,PrivateMessageManager.SORT_DESC);
+			  for (Iterator iter = decoratedPvtMsgs.iterator(); iter.hasNext();) {
```
Comment (ern) [inline msgcntr/messageforums-app/src/java/org/sakaiproject/tool/messageforums/PrivateMessagesTool.java]: Use newer for loop please its cleaner and less lines.

**Comment by juanjmerono** [comment]
Done !

**Comment by ottenhoff** [comment]
@ern can you re-review?

**Pull request merged**